src/search/indexing/mod.rs

```diff
@@ -129,9 +129,12 @@ async fn update_index<'a>(
 ) -> Result<Index<'a>, IndexingError> {
     let index = match client.get_index(name).await {
         Ok(index) => index,
-        Err(meilisearch_sdk::errors::Error::IndexNotFound) => {
-            client.create_index(name, Some("mod_id")).await?
-        }
+        Err(meilisearch_sdk::errors::Error::MeiliSearchError {
+            message,
+            error_code: meilisearch_sdk::errors::ErrorCode::IndexNotFound,
+            error_type,
+            error_link,
+        }) => client.create_index(name, Some("mod_id")).await?,
```

Comment:
This could be
```
Err(meilisearch_sdk::errors::Error::MeiliSearchError {
    error_code: meilisearch_sdk::errors::ErrorCode::IndexNotFound,
    ..
}
```
since none of the fields are used